backend/pkg/serde/protobuf_schema.go

```diff
@@ -102,9 +102,9 @@ func (d ProtobufSchemaSerde) DeserializePayload(ctx context.Context, record *kgo
 
 	// Marshal proto message into JSON
 	o := protojson.MarshalOptions{
-		UseProtoNames: true, // use snake_case
+		UseProtoNames: false, // use loweCamelCase
 		// Do not use EmitUnpopulated, so we don't emit nulls (they are ugly, and provide no benefit. they transport no information, even in "normal" json).
-		EmitUnpopulated: false,
+		EmitUnpopulated: true,
```

Comment:
These changes keep the functionality and behaviour same as is in current master

Reply:
Since we cut a new major release we would be allowed of doing so. Imo that would be a slight improvement over what we currently do, since it matches the proto field names more closely. WDYT?

Reply:
> since it matches the proto field names more closely. WDYT?

Regarding JSON field names and casing, the [default JSON representation](https://protobuf.dev/programming-guides/json/) _is_ `lowerCamelCase`. 

Similarly regarding emitting unpopulated fields. Default behaviour is to _omit_ those fields.

I do not feel strongly on this but part of me thinks it's going against more standard convention and defaults. 
I can be convinced otherwise especially if it's a new major release, it relaxes some of the compatibility issue I brought up.

Reply:
Okay fine for me to change as you suggested 👍